client/components/SingleArtwork.js

```diff
@@ -1,49 +1,69 @@
 import React from  'react';
 import { connect } from 'react-redux';
 import { getArtwork } from '../redux/artwork';
-import {Link} from 'react-router-dom'
-
+import { Link } from 'react-router-dom'
+import { addCartItem } from '../redux/cart'
 
 export class SingleArtwork extends React.Component {
+    constructor() {
+        super()
+        this.addQty = this.addQty.bind(this)
+    }
+
+    async addQty(){
+        const quantity = +document.getElementById('qty').value
```

Comment:
I think you should be able to get the ev.target.value instead of having to use getElementById here

Reply:
The event here would be the button press. I could store qty in the state, but this seemed simpler.